### PR TITLE
Implement object header loading

### PR DESCRIPTION
**Pull request by Felerius**

### CODE REVIEW

**Comment by lmm-git** [comment]
Nice work :+1: 

Please keep in mind to add always a point to the end of a function like we did in our previous project (also at getter and setter just to have clean outputs of SWALint).

File: packages/ETLoad.package/ETLObjectHeader.class/instance/classOop.st [hunk 1]
```diff
@@ -1,3 +1,5 @@
 accessing
 classOop
+	"Oop to the class of the object. Nil if the class is compacted."
```
Comment (TimNN) [inline packages/ETLoad.package/ETLObjectHeader.class/instance/classOop.st:3]: What does Oop stand for?

**Comment by TimNN** [comment]
This looks pretty good, great work!

I would prefer two changes regarding the binary literals:
- If they consist solely of trailing one's, write them as hexadecimal (I find `16r0FFF` and `16r1F` much easier to read than `2r111111111111` and `2r11111`, especially since the hex digits `1, 3, 7, F` are easily recognisable).
- Pad all binary and hexadecimal literals to whole bytes (so make sure you have an even number of hexadecimal digits and a multiple of 8 binary digits).

File: packages/ETLoad.package/ETLObjectHeader.class/instance/classOop.st (hunk 1 above)
Comment (Felerius) [inline packages/ETLoad.package/ETLObjectHeader.class/instance/classOop.st:3]: OOP is a abbreviation for "object oriented pointer", and is basically the same as a pointer or reference in other languages. The term OOP seems to be quite commonly used in the smalltalk world, which is why we chose to use it here. See the section "Basic OOP Concepts and Terminology" on [this page](http://web.cecs.pdx.edu/~harry/musings/SmalltalkOverview.html) for more info.

**Comment by Felerius** [comment]
Some comments:
- The reason we used all binary literals in the tests is so you can compare the extracted information with the input binary data. To make this easier we used recognizable binary patterns (like `10` repeated, or `1100` repeated). Unfortunately we can't use all hexadecimal numbers, because many header parts aren't aligned with 4 bit blocks.
- Same reason as above, the literals should be recognizable as parts of the binary input they're parsed from. Padding with zeros would lead to the literals not actually being a substring of the input any more. For any other use of binary/hexadecimal literals I would totally agree with you.

**Comment by Felerius** [comment]
@TimNN @lmm-git Updated based on feedback.

**Comment by TimNN** [comment]
Looks good, I'll merge this as soon as travis passes.

**Pull request merged**